### ISSUE TITLE
PDX Details page organization

### ISSUE THREAD

**Issue by CarolBult** [mgijax/mtb-wi#76]
@ahcurrier @sneuhaus 

1. The PDX Details pages should be organized as 4 sections (see attached screenshot):
* title
*model summary
*molecular data
*model characterization

2. The font sizes for section headers, data type labels within a section, data, and explanatory notes should be different sizes and/or styles and consistently implemented across the sections on the page.

3. Details on how each section should look are detailed in individual GitHub tickets for each section. 

<img width="1429" alt="Screen Shot 2020-08-18 at 9 29 28 AM" src="https://user-images.githubusercontent.com/8211639/90518907-598ab400-e135-11ea-9e2c-01c1526b023a.png">

**Comment by ahcurrier** [comment]
Thanks for splitting this up into smaller issues, @CarolBult . This looks like a new development project, since it changes how the page is structured compared to the old site. Is this a correct assessment?

**Comment by CarolBult** [comment]
The general organizational paradigm: model summary, molecular details, model characterization is not new and some of this organization got lost in translation.

However there are other elements that could legitimately be considered new development ---such as fixing the location of the fields within a ribbon and moving all of the molecular annotations into the molecular summaries section

CB

--
Carol J. Bult, Ph.D.
Professor and Knowlton Family Chair
The Jackson Laboratory
600 Main Street
RL13
Bar Harbor, Maine 04609
(tel) 207-288-6324
(email) carol.bult@jax.org<mailto:carol.bult@jax.org>
